modules/tensor_mechanics/include/materials/ADLAROMANCEStressUpdateBase.h

```diff
@@ -252,8 +269,35 @@ class ADLAROMANCEStressUpdateBase : public ADRadialReturnCreepStressUpdateBase
   /// Container for old immobile dislocation value
   Real _immobile_old;
 
+  /// Container for old effective strain
+  Real _effective_strain_old;
+
+  /// Index corresponding to the position for the mobile disloations in the input vector
+  const unsigned int _mobile_input_idx;
+
+  /// Index corresponding to the position for the immobile disloations in the input vector
+  const unsigned int _immobile_input_idx;
```

Comment:
I had thought that Aaron wanted these to be a vector /meta-string/ thingambober that would give the positions of the inputs and outputs. Did you discuss that option and elect this set of indices instead?

Reply:
I'm not aware of that, maybe we can chat about that this afternoon? This should achieve the same purpose no?